scripts/pretest.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+# interrupt on any error
+set -e
+
+# load .env file to get NODE_ENV
+if [ -f ".env" ]; then
+  source ".env"
+fi
+
+npm run hint
+
+if [ "$NODE_ENV" = "circleci" ] || [ "$NODE_ENV" = "development" ]; then
+  npm run rexec:api install_selenium_server.sh
```

Comment:
`install_selenium_server.sh` should be moved to `app/scripts/`.

I'd get rid of that file and move it to `package.json`